src/tests/ftest/util/data_mover_utils.py

```diff
@@ -5,19 +5,132 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
 
-
 from command_utils_base import FormattedParameter
 from command_utils_base import BasicParameter
 from command_utils import ExecutableCommand
 from job_manager_utils import Mpirun
 
+def format_daos_path(pool=None, cont=None, path=None):
+    """Format a daos path as daos://<pool>/<cont>/<path>.
+
+    Args:
+        pool (TestPool, optional): the source pool.
+            Alternatively, this can be the pool uuid.
+        cont (TestContainer, optional): the source container.
+            Alternatively, this can be the container uuid.
+        path (str, optional): The posix-style path relative
+            to the container root.
+
+    Returns:
+        str: The formatted path.
+    """
+    daos_path = "daos://"
+    if pool:
+        pool_uuid = DataMoverTestBase._uuid_from_obj(pool)
+        daos_path += str(pool_uuid) + "/"
+    if cont:
+        cont_uuid = DataMoverTestBase._uuid_from_obj(cont)
```

Comment:
(pylint-protected-access) Access to a protected member _uuid_from_obj of a client class
(pylint-undefined-variable) Undefined variable 'DataMoverTestBase'